oai_agents/common/tags.py

```diff
@@ -60,7 +67,8 @@ def map_to_index(teamtype):
             TeamType.SELF_PLAY: 9,
             TeamType.SELF_PLAY_LOW: 10,
             TeamType.SELF_PLAY_MEDIUM: 11,
-            TeamType.SELF_PLAY_HIGH: 12
+            TeamType.SELF_PLAY_HIGH: 12,
+            TeamType.ALL_MIX: 13    # TODO: Should this go here or after HLR?
```

Comment:
Is this in the right spot for the `map_to_index`?

Reply:
yes